### DISCUSSION TITLE
Latest version = 0.28.0?

### DISCUSSION THREAD

**Discussion by JP-CC-Projects** [axios/axios#6230] [General]
Hi Devs, I was wondering why the latest version is .28. I am wondering if making this the latest main branch was on purpose. Installing the latest could break some code, as exampled in this screenshot. It took 2 seconds for me to fix and maybe this is a non-issue but wanted to point out.

![image](https://github.com/axios/axios/assets/129788516/e64d1eed-b4d0-4c55-b635-d9283832fd79)

**Comment by jasonsaayman** [comment]
No a number newer ones have been released. Check the releases for the latest.
